### PR TITLE
Base for TriggersCollection ✨

### PR DESCRIPTION
**Pull request by gregorylegarec**
This PR introduces base methods for creating and launching a trigger.

Status is WIP until I successfully use them in Cozy-Harvest-Lib.

### CODE REVIEW

File: packages/cozy-stack-client/src/TriggerCollection.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { normalizeDoc } from './DocumentCollection'
+import { uri } from './utils'
+
+export const JOBS_DOCTYPE = 'io.cozy.triggers'
```
Comment (Crash--) [inline packages/cozy-stack-client/src/TriggerCollection.js]: io.cozy.jobs ?

File: packages/cozy-stack-client/src/TriggerCollection.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    this.stackClient = stackClient
+  }
+
+  async all() {
+    throw new Error('all() method is not yet implemented')
+  }
+
+  /**
+   * Creates a Trigger document
+   * @see https://github.com/cozy/cozy-stack/blob/master/docs/jobs.md#post-jobstriggers
```
Comment (Crash--) [inline packages/cozy-stack-client/src/TriggerCollection.js]: Should we use doc.cozy.io ?

Comment (gregorylegarec) [reply packages/cozy-stack-client/src/TriggerCollection.js]: Yeah, probably, I think.

**Comment by gregorylegarec** [comment]
No more in WIP ✨

File: packages/cozy-stack-client/src/TriggerCollection.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  async launch(trigger) {
+    const path = uri`/jobs/triggers/${trigger._id}/launch`
+    const resp = await this.stackClient.fetchJSON('POST', path)
+    return {
+      data: normalizeJob(resp.data)
+    }
+  }
+
+  async update() {
+    throw new Error('update() method is not available for applications')
```
Comment (CPatchane) [inline packages/cozy-stack-client/src/TriggerCollection.js]: s/for applications/for triggers ?

**Pull request merged**